public/external-text/default.yaml

```diff
@@ -51,15 +51,15 @@ help:
     items:
       - question: Which model should I select?
         answer: |
-          Canada's own General Circulation Model, CanESM2, is the most representative
+          Canada's own General Circulation Model, CanESM2/CanESM5 (CMIP5/CMIP6 respectively), is the most representative
           model for Canada. It is a good default choice if you do not have
           specialized needs that would be better represented by a different model.
 
       - question: What emissions scenario should I select?
         answer: |
-          The worst-case emissions scenario, RCP 8.5
-          (presented in the selectors as `historical, rcp85`),
-          is a good choice for making decisions that adapt to climate change.
+          The worst-case emissions scenarios, RCP 8.5 and SSP 5-8.5
+          (presented in the selectors as `historical, rcp85` and `historical, ssp585` for CMIP5 and CMIP6 respectively),
```

Comment:
Are they not presented as "Historical, then RCP 8.5" and "Historical, then SSP 5-8.5"? Wasn't that the point of the update to pcic-react-components?

Reply:
Yes. I just kept this format because `historical, rcp85` was used in the old `default.yaml` file. But I'll change it to the format in pcic-react-components.